sdk/python/tests/integration/feature_repos/universal/data_source_creator.py

```diff
@@ -11,12 +11,26 @@ class DataSourceCreator(ABC):
     @abstractmethod
     def create_data_source(
         self,
-        destination: str,
         df: pd.DataFrame,
+        destination: str,
         event_timestamp_column="ts",
         created_timestamp_column="created_ts",
         field_mapping: Dict[str, str] = None,
     ) -> DataSource:
+        """
+        Create a data source based on the dataframe. Implementing this method requires the underlying implementation to
+        persist the dataframe in offline store, using the destination string as a way to differentiate multiple
+        dataframes and data sources.
+
+        :param df: The dataframe to be used to create the data source.
```

Comment:
Nitpick: These are not Google style docstrings. Btw you can change your PyCharm to automatically generate the correct format.